### PR TITLE
Update Entity Framework tutorial for version 11

### PR DESCRIPTION
**Pull request by jewpaltz**
Just wanted to update the blog post introducing the integration with Entity Framework, because that was a major confusion point for me.
I searched for information about using HotChocolate and Entity Framework. All I could find was information from version 10 or https://github.com/ChilliCream/graphql-workshop which was way too much of an overkill.

I hope you find this helpful.

### CODE REVIEW

File: website/src/blog/2020-03-18-entity-framework/2020-03-18-entity-framework.md [hunk 1]
```diff
@@ -627,29 +544,52 @@ SELECT "s"."FirstMidName",
...
 
 Let us go further with this. We actually can do more here and Hot Chocolate provides you with a filter and sorting middleware to really give you the power to query your data with complex expressions.
 
-First, we need to add two more packages that will add the sorting and filtering middleware.
+First, we need to add filtering and sorting providers to our GraphQL service.
 
 ```bash
 dotnet add package HotChocolate.Types.Filters
 dotnet add package HotChocolate.Types.Sorting
 ```
```
Comment (PascalSenn) [inline website/src/blog/2020-03-18-entity-framework/2020-03-18-entity-framework.md:563]: I think we can also remove this :)

File: website/src/blog/2020-03-18-entity-framework/2020-03-18-entity-framework.md [hunk 2]
```diff
@@ -627,29 +544,52 @@ SELECT "s"."FirstMidName",
...
 
 With these new packages in place let us rewrite our query type in order to enable proper filtering support.
+In `Startup.cs` add the following two lines right after `.AddQueryType<>()`
```
Comment (PascalSenn) [inline website/src/blog/2020-03-18-entity-framework/2020-03-18-entity-framework.md:565]:
```suggestion
In `Startup.cs` add the following two lines  after `.AddGraphQLServer()`
```

**Review by PascalSenn — CHANGES_REQUESTED** [review]
Thank you for this PR!! This will help the community a lot!

File: website/src/blog/2020-03-18-entity-framework/2020-03-18-entity-framework.md (hunk 2 above)
Comment (PascalSenn) [reply website/src/blog/2020-03-18-entity-framework/2020-03-18-entity-framework.md:565]: FYI: Order is not important in this case :)

File: website/src/blog/2020-03-18-entity-framework/2020-03-18-entity-framework.md (hunk 1 above)
Comment (jewpaltz) [reply website/src/blog/2020-03-18-entity-framework/2020-03-18-entity-framework.md]: Of course!
Sorry for the oversight.

**Pull request closed without merging**